project/Settings.scala

```diff
@@ -82,14 +83,14 @@ object Settings {
     val jaxb                = "2.3.1"
 
     // Gemini Libraries
-    val lucumaCore    = "0.15.1"
-    val lucumaUI      = "0.20.0"
-    val lucumaSchemas = "0.1.7"
+    val lucumaCore    = "0.45.0"
```

Comment:
I guess this version of core will get fossilized on 0.45.0 for scala 2, until we can switch to scala 3